harvester/projects/models/resources/buas.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+from django.conf import settings
+
+from datagrowth.resources import HttpResource
+
+
+logger = logging.getLogger("harvester")
+
+
+class BuasPureProjectResource(HttpResource):
```

Comment:
Deze class declareert hoe het systeem moet omgaan met de BUAS API. Alle API responses worden opgeslagen in de database, zodat bugs makkelijker te debuggen zijn. Bovendien doet herhaaldelijk "harvesten" dus niet altijd echte requesten. Waar mogelijk en waar toegestaan zal het systeem API responses herbruiken. Dit loopt allemaal via de HttpResource interface.